packages/db/src/async_operations.ts

```diff
@@ -0,0 +1,33 @@
+import { toSnake } from 'snake-camel';
+
+interface CamelCaseAsyncOperation {
+  id: string
+  description: string
+  operationType: string
+  status: string
+  output?: object
+  taskArn?: string
+  createdAt?: Date
+  updatedAt?: Date
+}
+
+interface SnakeCaseAsyncOperation {
```

Comment:
I think this should be RDSAsyncOperation or PostgresAsyncOperation

Reply:
I thought about doing that, but I think that gives it a context of officiality that I'm trying to avoid bestowing on these interfaces, as I want them to be flexible and their only relevant property is the property casing.

Reply:
I don't agree. The functional use of these interfaces is for distinguishing data types we handle that are either coming from/going to Dynamo and going into Postgres. Naming them based on the casing doesn't make that clear.

Reply:
Although the motivating factor for the creation of these interfaces is that, I have two concerns:
1. I don't want to confuse our typing scheme any more than necessary because we are already carrying one type each for RDS and Dynamo in the `db` and `types` packages, respectively. Anybody looking at this for the first time is going to have a hard time understanding what the intended difference is between a `db/types.AsyncOperationRecord` and a `db/async_operations.RDSAsyncOperation`. I can probably comment extensively instead, if that's preferred.
2. The action of this function is to translate a record's column names to snake case. We are doing this as a side effect of RDS requirements, but that doesn't, in my view, make this a 'convert to perfect RDS format' function, as it doesn't do the kinds of validation that would require. Given that, `db` might be a bad place for the translator, or you could say we need a more extensive function. I'm interested in your thoughts here.

Reply:
As a corollary to no. 2, currently I would argue the types should not be concerned with RDS vs. Dynamo, as the function is not concerned with RDS vs. Dynamo. It is only concerned with top-level object keys and their casing.